llvm/lib/CodeGenData/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+add_llvm_component_library(LLVMCodeGenData
```

Comment:
Where are you using this library? It seems to only be defined and only used in the test.

Reply:
yeah. It is currently used only in the test, but the subsequent PRs will use it.